google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -454,6 +454,9 @@ managedsdk.progress.message=Managed Google Cloud SDK: {0}.
 managedsdk.progress.install.app.engine=Installing App Engine Java
 managedsdk.progress.update=Updating Google Cloud SDK
 managedsdk.waiting.for.sdk.ready=Waiting for Google Cloud SDK installation to complete...
+managedsdk.update.notification=New version of Managed Google Cloud SDK is available and about to be updated.
+managedsdk.update.notification.cancel=Cancel this update
```

Comment:
is it "Cancel" or "Skip"?

Reply:
Skip probably describes it better, let me rename.